spec/models/spree/taxon_spec.rb

```diff
@@ -2,102 +2,154 @@
 
 require 'spec_helper'
 
-module Spree
-  RSpec.describe Taxon do
-    let(:taxon) { Spree::Taxon.new(name: "Ruby on Rails") }
-
-    let(:e) { create(:supplier_enterprise) }
-    let(:t1) { create(:taxon) }
-    let(:t2) { create(:taxon) }
-
-    describe "finding all supplied taxons" do
-      let!(:p1) {
-        create(:simple_product, primary_taxon_id: t1.id, supplier_id: e.id)
-      }
+RSpec.describe Spree::Taxon do
```

Comment:
These are mostly just whitespace changes in this file (changed from `module Spree` nesting to `RSpec.describe Spree::Taxon` to avoid linting problems) - I would recommend hiding whitespace to view, to distinguish the functional differences